app/views/users/_profile.html.erb

```diff
@@ -9,12 +9,24 @@
 
     <div class="crayons-field">
       <%= f.label :name, for: "user[name]", class: "crayons-field__label" %>
-      <%= f.text_field "user[name]", maxlength: 30, class: "crayons-textfield", placeholder: "John Doe", value: @user.name %>
+      <%= f.text_field "user[name]",
+                       maxlength: 30,
+                       class: "crayons-textfield",
+                       placeholder: "John Doe",
+                       value: @user.name,
+                       data: { character_span_id: "name-characters" } %>
+      <div class="align-self-flex-end fs-s"><span id="name-characters"></span>/30</div>
     </div>
 
     <div class="crayons-field">
       <%= f.label :email, for: "user[email]", class: "crayons-field__label" %>
-      <%= f.text_field "user[email]", class: "crayons-textfield", placeholder: "john.doe@example.com", value: @user.email %>
+      <%= f.text_field "user[email]",
+                       maxlength: 50,
+                       class: "crayons-textfield",
+                       placeholder: "john.doe@example.com",
+                       value: @user.email,
+                       data: { character_span_id: "email-characters" } %>
+      <div class="align-self-flex-end fs-s"><span id="email-characters"></span>/50</div>
```

Comment:
Accessibility (potentially blocking): If this is something that needs to be communicated to someone blind or visually impaired, we'll need to add the `aria-live="polite"` attribute/value pair.

I'll defer to @aitchiss, but since there is a `maxlength` attribute on these fields, an `aria-hidden="true"` might be more appropriate.

Reply:
I think probably the best way to handle this for accessibility would be to link this information via `aria-describedby` - although we have the maxlength field set, screen reader users won't be aware of the character limitation until they're preventing from typing further.

I think we could link everything together by changing this line (and similar to):
```suggestion
      <div id="email-description" class="align-self-flex-end fs-s"><span class="screen-reader-only">Available characters used: </span><span id="email-characters"></span>/50</div>
```

and then adding `aria-describedby="email-description"` to the input element.

I would file this under "suggested enhancement" though, so please consider non-blocking for this PR